### ISSUE TITLE
Move astroquery import into scrub module subclass

### ISSUE THREAD

**Issue by stscijgbot-jp** [spacetelescope/jwst#8407]
_Issue [JP-3591](https://jira.stsci.edu/browse/JP-3591) was created on JIRA by [Ru Kein](https://jira.stsci.edu/secure/ViewProfile.jspa?name=rkein):_

spacekit.skopes.jwst.cal.predict imports spacekit.preprocessor.scrub to run preprocessing on datasets using the JwstCalScrubber subclass. Another subclass inside the scrub module, unrelated to JWST, relies on the radio.py module, which uses a try/except to import astroquery, an extra dependency not needed or intended to be used by JWST Cal Pipeline. The expected behavior is for the astroquery import to be ignored for JWST preprocessing. However, it turns out that astroquery is included as a dependency in SDP's requirements.txt, so the import succeeds when loading the scrub module. The issue is that astroquery apparently creates a folder `~/.astropy/cache/astroquery/Observations/` and if we have more than one estimation happening at the same time it leads to "FileExistsError".  Full traceback is below.

As a workaround, Eric Barron is checking to see where SDP might be using astroquery. He mentioned that while it is in the requirements.txt, that could be due to it just being pulled in while installing something else and it might be possible to remove it.

This error specifically occurs when attempting to run multiple estimations at once and should be considered a bug. The resolution should be to avoid importing astroquery except by the specific subclass that requires it (HstSvmScrubber).

 
```java
Traceback (most recent call last):
  File "/ifs/archive/dev/jwst/store/dmsauto/Nightly_tests/test_dmsauto_20240329_01/install/jwst_ops/bin/estimate_level_3_memory.py", line 25, in <module>
    from spacekit.skopes.jwst.cal.predict import JwstCalPredict
  File "/ifs/pipeline/dms/rhel7/pkgs/miniconda3-jwst/envs/jwowl-2024.1.0/lib/python3.11/site-packages/spacekit/skopes/jwst/cal/predict.py", line 16, in <module>
    from spacekit.preprocessor.scrub import JwstCalScrubber
  File "/ifs/pipeline/dms/rhel7/pkgs/miniconda3-jwst/envs/jwowl-2024.1.0/lib/python3.11/site-packages/spacekit/preprocessor/scrub.py", line 12, in <module>
    from spacekit.extractor.radio import HstSvmRadio
  File "/ifs/pipeline/dms/rhel7/pkgs/miniconda3-jwst/envs/jwowl-2024.1.0/lib/python3.11/site-packages/spacekit/extractor/radio.py", line 10, in <module>
    from astroquery.mast import Observations
  File "/ifs/pipeline/dms/rhel7/pkgs/miniconda3-jwst/envs/jwowl-2024.1.0/lib/python3.11/site-packages/astroquery/mast/__init__.py", line 37, in <module>
    from .core import Observations, ObservationsClass, Catalogs, CatalogsClass, Mast, MastClass
  File "/ifs/pipeline/dms/rhel7/pkgs/miniconda3-jwst/envs/jwowl-2024.1.0/lib/python3.11/site-packages/astroquery/mast/core.py", line 2336, in <module>
    Observations = ObservationsClass()
                   ^^^^^^^^^^^^^^^^^^^
  File "/ifs/pipeline/dms/rhel7/pkgs/miniconda3-jwst/envs/jwowl-2024.1.0/lib/python3.11/site-packages/astroquery/mast/core.py", line 1045, in __init__
    super(ObservationsClass, self).__init__(*args, **kwargs)
  File "/ifs/pipeline/dms/rhel7/pkgs/miniconda3-jwst/envs/jwowl-2024.1.0/lib/python3.11/site-packages/astroquery/mast/core.py", line 269, in __init__
    super(MastClass, self).__init__()
  File "/ifs/pipeline/dms/rhel7/pkgs/miniconda3-jwst/envs/jwowl-2024.1.0/lib/python3.11/site-packages/astroquery/query.py", line 394, in __init__
    super(QueryWithLogin, self).__init__()
  File "/ifs/pipeline/dms/rhel7/pkgs/miniconda3-jwst/envs/jwowl-2024.1.0/lib/python3.11/site-packages/astroquery/query.py", line 159, in __init__
    os.makedirs(self.cache_location)
  File "<frozen os>", line 225, in makedirs
FileExistsError: [Errno 17] File exists: '/ifs/archive/dev/jwst/store/dmsauto/Nightly_tests/test_dmsauto_20240329_01/regr_10011/defs/sdp_python/astropy/astroquery/Observations' ```

**Comment by stscijgbot-jp** [comment]
_Comment by [Ru Kein](https://jira.stsci.edu/secure/ViewProfile.jspa?name=rkein) on [JIRA](https://jira.stsci.edu/browse/JP-3591?focusedCommentId=692247#comment-692247):_

So far I'm unable to reproduce the error.  The only way I can get it to create the problematic cache path in question is by directly importing the Observations module. In any case there is an attempted fix for this in spacetelescope/spacekit#49

**Comment by stscijgbot-jp** [comment]
_Comment by [Ru Kein](https://jira.stsci.edu/secure/ViewProfile.jspa?name=rkein) on [JIRA](https://jira.stsci.edu/browse/JP-3591?focusedCommentId=692537#comment-692537):_

Fix version for spacekit: 1.0.1

Released on pypi: <https://pypi.org/project/spacekit/>
